### PR TITLE
Makes mixing meth dangerous again

### PR DESCRIPTION
**Pull request by lordpidey**
[Changelogs]: Reverts #29912 while fixing the same problem it solves in a better way.

[why]: Mixing meth is supposed to be dangerous and potentially explosive if done wrong.  Though methsplosion syringes did need to be removed, there was no need to make meth less explosive, we just needed to make the explosions not happen inside of mobs.

### CODE REVIEW

**Comment by Ninja137** [comment]
Entirely blacklist it from happening in mobs, but make fucking it up actually fuck you up.

**Comment by lordpidey** [comment]
It is entirely blacklisted from happening in mobs.

I'm hesitant to make the explosion worse, otherwise we'll see it as a grenade mix.

File: code/modules/reagents/chemistry/recipes/pyrotechnics.dm [hunk 1]
```diff
@@ -140,10 +140,11 @@
 	name = "Meth explosion"
 	id = "methboom1"
 	results = list("methboom1" = 1)
-	required_temp = 420 //high enough to not blow up in the meth syringes
+	required_temp = 380 //slightly above the meth mix time.
 	required_reagents = list("methamphetamine" = 1)
 	strengthdiv = 6
 	modifier = 1
+	mob_react = TRUE //Yes, setting it true makes it NOT react in mobs.
```
Comment (ChangelingRain) [inline code/modules/reagents/chemistry/recipes/pyrotechnics.dm]: please fix this fucking varname

**Comment by Ninja137** [comment]
I mean, you can already get ridiculous grenades through a variety of means. Acid-lube-singulo nades that pull everything in and start a death-cascade, CLF3-Lube-Singulo grenades, massive Potassium+Water, Tesla shock grenades, and all sorts of other shit.

**Comment by BeeSting12** [comment]
Thanks, this does it a lot better than my PR.

**Comment by lordpidey** [comment]
Aright, I've spent a few hours messing around with chemistry, and found out that it's not as viable of a grenade ingredient as I thought, as there's no way to make methsplosion smoke or methsplosion foam.

Because of this, I'm up for re-considering increasing the explosion power.

**Comment by ExcessiveUseOfCobblestone** [comment]
a meth explosion grenade doesnt sound that good in comparison to other grenade options that are also easier to make

**Comment by TimeIsEndless** [comment]
I understand the change to the meth but why change all those other reactions?

**Comment by lordpidey** [comment]
The mob_react variable, if true, prevents the reaction from occurring in a mob.

That's unintuitive, so I fixed it, and thus I had to reverse literally every instance of that variable.

**Pull request merged**